.github/workflows-source/ci-main.yml

```
@@ -152,8 +152,8 @@ jobs:
           RUN_ON_DIFF_ONLY: false
       - <<: *bazel-bep
 
-  bazel-test-darwin-x86-64:
-    name: Bazel Test Darwin x86-64
+  bazel-test-macos-intel:
+    name: Bazel Test MacOS Intel
```

```suggestion
    name: Bazel Test macOS Intel
```

I think the correct capitalization is `macOS` not `MacOS`